.cbmc-batch/jobs/aws_cryptosdk_sig_get_pubkey/aws_cryptosdk_sig_get_pubkey_harness.c

```diff
@@ -24,13 +24,11 @@
 
 void aws_cryptosdk_sig_get_pubkey_harness() {
     /* arguments */
-    struct aws_cryptosdk_sig_ctx *ctx = can_fail_malloc(sizeof(struct aws_cryptosdk_sig_ctx));
+    struct aws_cryptosdk_sig_ctx *ctx = ensure_nondet_sig_ctx_has_allocated_members();
     struct aws_allocator *alloc       = can_fail_allocator();
     struct aws_string *pubkey;
```

Comment:
You need to properly allocate this string here.
```c
struct aws_string *pubkey = malloc(sizeof(*pubkey));
```
And use the pointer when calling `aws_cryptosdk_sig_get_pubkey`; otherwise, you missed the case where it's null.